src/components/testimonials.jsx

```diff
@@ -12,7 +12,7 @@ export const Testimonials = (props) => {
                   <div className="testimonial">
                     <div className="testimonial-image">
                       {" "}
-                      <img src={d.img} alt="" />{" "}
+                      <img src={d.img} alt="testimonial image" />{" "}
```

Comment:
nit: I think putting "picture", "image" or "photo" in an <img> tag is not great because these screen readers (for blind users) automatically attaches images when reading this. Let's do "Testimonial headshot" instead here

Reply:
Thanks for the feedback - will take another look.